README.md

```diff
@@ -0,0 +1,73 @@
+o-meter [![Circle CI](https://circleci.com/gh/Financial-Times/o-meter/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-meter/tree/master)[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
+=================
+
+Use the meter element to measure data within a given range. The <meter> tag defines a scalar measurement within a known range, or a fractional value. This is also known as a gauge.
+
+- [Markup](#markup)
+- [Sass](#sass)
+- [Support](#support)
+- [Contact](#contact)
+- [Licence](#licence)
+
+### Markup
+
+This HTML demonstrates a way to use a basic o-meter
+
+```html
+<meter class="o-meter" data-o-component="o-meter" min="0" max="100" value="25">
+	25
+</meter>
+```
+
+This HTML demonstrates a way to use an extended o-meter with an additional value indicator
+```html
+<div class="o-meter-container">
+	<meter class="o-meter" data-o-component="o-meter" min="0" max="100" value="75">
+	75
+	</meter>
+	<span class="o-meter-value" style="left: 75%">
+		75
+	</span>
+</div>
+```
+This HTML demonstrates a way to use a basic o-meter with customised colours
+```html
+	<meter class="o-meter" style="
+		--o-meter-background-color: hotpink;
+		--o-meter-optimum-color: deeppink;
+		--o-meter-sub-optimum-colo: pink;
+		--o-meter-sub-sub-optimum-color: red;" data-o-component="o-meter" min="0" max="100" value="{{meterValue}}">
```

Comment:
I love that Firefox went with `sub-sub-optimum` and webkit went with `even-less-good` naming for their CSS selector naming 😂 Maybe we should name our CSS variables after the [HTML attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter) for these thresholds instead? There is `low`, `high`, `optimum`, we could go with: `--o-meter-low-color`, `--o-meter-high-color`, `--o-meter-optimum-color`

Reply:
agree, those namings are amusing, indeed. I will change it